test/integration/narrow/helpers/other-items-sizes.test.js

```diff
@@ -658,8 +658,8 @@ describe('Other Items Sizes Array Function', () => {
           {
             type: 'MIN_MAX',
             min: 1,
-            max: 9999,
-            error: 'Quantity must be between 1-9999'
+            max: 999999,
+            error: 'Quantity must be between 1-999999'
```

Comment:
![image](https://github.com/DEFRA/ffc-grants-slurry-web/assets/15157025/f4e2cff3-4bbf-423b-8f3f-436636b436b4)
Agitators should be 1-9999